lib/launch-fastify.js

```diff
@@ -22,12 +22,17 @@ const fastifySwagger = require('fastify-swagger')
 const fastifySensible = require('fastify-sensible')
 const absolutePath = require('./absolute-path')
 const customLogger = require('./custom-logger')
+const { logIncomingRequest, logRequestCompleted } = require('./custom-logger')
 const importEnv = require('./import-env')
 const { exportFastifyOptions, exportServiceOptions } = require('./options-extractors')
 const statusRoutes = require('./status-routes')
 const swaggerDefinition = require('./swagger-definition')
 const WAIT_BEFORE_SERVER_CLOSE_SEC = parseInt(process.env.WAIT_BEFORE_SERVER_CLOSE_SEC) || 10
 
+const lc39Options = {
+  disableRequestLogging: true,
```

Comment:
We lost an error log with this.

Reply:
You mean this one?

https://github.com/fastify/fastify/blob/667d4644eab09f27bc30cad6e7378f5fc82cb9e8/lib/reply.js#L587